### PR TITLE
TodoMVC jQuery: fix clear completed button

### PR DESCRIPTION
**Pull request by flashdesignory**
Switched 'id' to 'class' for 'clear-completed'.
This is necessary for the css rules to apply correctly.

Fixes issue https://github.com/WebKit/Speedometer/issues/363

@kara

### CODE REVIEW

File: resources/todomvc/architecture-examples/jquery/package.json
Comment (flashdesignory) [inline resources/todomvc/architecture-examples/jquery/package.json:5]: not a fix here, but to be consistent with other framework scripts, added it here...

Comment (julienw) [reply resources/todomvc/architecture-examples/jquery/package.json:5]: yeah that's useful :-) thanks

**Review by julienw — APPROVED** [review]
Thanks for the quick turnaround!
Do you think you can wait for #360 to land before landing this one, to avoid that @issackjohn has to rebase the big patch? (even though I think there wouldn't be a conflict in this case)

**Comment by flashdesignory** [comment]
> Thanks for the quick turnaround! Do you think you can wait for #360 to land before landing this one, to avoid that @issackjohn has to rebase the big patch? (even though I think there wouldn't be a conflict in this case)

Thanks - yes, this can wait 😄

**Review by issackjohn — APPROVED** [review]

**Review by rniwa — APPROVED** [review]

**Review by camillobruni — APPROVED** [review]

**Pull request merged**
